src/pages/signin/SignInHeroImage.js

```diff
@@ -1,32 +1,24 @@
-import PropTypes from 'prop-types';
 import React from 'react';
 import {View} from 'react-native';
 import Lottie from '@components/Lottie';
 import LottieAnimations from '@components/LottieAnimations';
-import withWindowDimensions, {windowDimensionsPropTypes} from '@components/withWindowDimensions';
 import useIsSplashHidden from '@hooks/useIsSplashHidden';
+import useResponsiveLayout from '@hooks/useResponsiveLayout';
 import useThemeStyles from '@hooks/useThemeStyles';
+import useWindowDimensions from '@hooks/useWindowDimensions';
 import variables from '@styles/variables';
 
-const propTypes = {
-    ...windowDimensionsPropTypes,
-
-    shouldShowSmallScreen: PropTypes.bool,
-};
-
-const defaultProps = {
-    shouldShowSmallScreen: false,
-};
-
-function SignInHeroImage(props) {
+function SignInHeroImage() {
     const styles = useThemeStyles();
+    const {isMediumScreenWidth} = useWindowDimensions();
```

Comment:
NAB for this PR, but we should update `useResponsiveLayout` to export `isMediumScreenWidth` as well.